File: ios/CoffeeBreak/Models/AppConfig.swift
import Foundation

enum AppConfig {
    /// Base URL of the Railway-hosted backend. Replace once the backend is
    /// deployed (see backend/README.md). Point this at the Railway staging
    /// URL for Debug builds and production URL for Release once both exist.
    static let apiBaseURL = URL(string: "https://coffee-break-backend.up.railway.app")!
}

File: mobile/lib/models/app_config.dart
/// Base URL of the Railway-hosted backend. Replace once the backend is
/// deployed (see backend/README.md). Point this at the Railway staging
/// URL for debug builds and the production URL for release once both exist.
class AppConfig {
  AppConfig._();

  static const String apiBaseUrl = 'https://coffee-break-backend.up.railway.app';
}
